### PR TITLE
PEP8 improvements

### PR DESCRIPTION
**Pull request by krysros**
Fixes some code style warnings / errors generated by flake8

### CODE REVIEW

File: cheroot/test/test_conn.py [hunk 1]
```diff
@@ -8,8 +8,12 @@
 import six
 import pytest
 
-from cheroot._compat import HTTPConnection, HTTPSConnection, NotConnected, BadStatusLine
-from cheroot._compat import ntob, urlopen
+from cheroot._compat import HTTPConnection
```
Comment (webknjaz) [inline cheroot/test/test_conn.py]: What is the motivation for this change?

File: cheroot/test/test_core.py [hunk 2]
```diff
@@ -316,25 +316,29 @@ def testSlashes(self):
         # Make sure GET params are preserved.
```
Comment (webknjaz) [inline cheroot/test/test_core.py]: Changes of this module are potentially useless, since in contains tests related to cherrypy, which are likely to be removed

File: cheroot/test/test_conn.py (hunk 1 above)
Comment (krysros) [inline cheroot/test/test_conn.py]: 1. Imports should usually [be on separate lines](https://www.python.org/dev/peps/pep-0008/#imports).
2. The length of the line 11 [exceeds 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

File: cheroot/test/test_core.py (hunk 2 above)
Comment (krysros) [inline cheroot/test/test_core.py]: OK

File: cheroot/test/test_conn.py (hunk 1 above)
Comment (webknjaz) [inline cheroot/test/test_conn.py]: From the link you provided:
> It's okay to say this though:
> 
> from subprocess import Popen, PIPE

so for `from ... import ...` it's OKay to list a few of them

they may be split into several lines using several `from ... import ...` statements or just multilined like
```python
from some_lib import (
    importable1, importable2,
    importable3, importable4,
    etc,
)
```

Comment (webknjaz) [inline cheroot/test/test_conn.py]: Regarding 79 chars limitation. Nowadays people usually set higher threshold, as we don't have 80 char terminals limitation today. Still it's a good practice to avoid long lines to make code better readable.

File: cheroot/wsgi.py [hunk 3]
```diff
@@ -53,9 +53,11 @@ def __init__(self, bind_addr, wsgi_app, numthreads=10, server_name=None,
         self.request_queue_size = request_queue_size
         self.timeout = timeout
         self.shutdown_timeout = shutdown_timeout
-        self.requests = threadpool.ThreadPool(self, min=numthreads or 1, max=max,
+        self.requests = threadpool.ThreadPool(
+            self, min=numthreads or 1, max=max,
             accepted_queue_size=accepted_queue_size,
-            accepted_queue_timeout=accepted_queue_timeout)
+            accepted_queue_timeout=accepted_queue_timeout
+            )
```
Comment (webknjaz) [inline cheroot/wsgi.py]: I think this paren should be one tab left, according to PEP8

File: cheroot/test/test_conn.py (hunk 1 above)
Comment (krysros) [inline cheroot/test/test_conn.py]: I agree.
